recipes-support/softhsm-testtoken/files/createtoken.sh

```diff
@@ -13,7 +13,8 @@ fi
 mkdir -p /var/lib/softhsm/tokens
 softhsm2-util --init-token --slot 0 --label "Virtual token" --pin 1234 --so-pin 1234
 
-softhsm2-util --import /var/sota/import/pkey.pem --label "pkey" --id 02 --token 'Virtual token' --pin 1234
+openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in /var/sota/import/pkey.pem -out /var/sota/import/pkey.p8
+softhsm2-util --import /var/sota/import/pkey.p8 --label "pkey" --id 02 --token 'Virtual token' --pin 1234
```

Comment:
What was the story here? I can't tell why this change was necessary.

Reply:
On master `shofthsm2-util --import` would fail on pkcs#1 keys. I'm not sure how it worked without this change, but it didn't work for me.

Reply:
Strange. I never had a problem. Was there something different about how you generate the keys with the implicit CA recipe? It's not a big deal, but it seems odd that we need the interim step.

Reply:
I can't see any differences. It might be because of different openssl versions. Maybe. But it looks weird that it worked before, because we've already had exactly the same problems with Uptane keys (see below in the same file).

Reply:
I'd like to understand the differences, but that's something we can come back to at a later time. Not worth holding back the PR just for that.